pkgs/top-level/python-packages.nix

```diff
@@ -23151,6 +23151,26 @@ in {
     };
   };
 
+  typeguard = buildPythonPackage rec {
```

Comment:
There are test, but they are probably not available on pypi. If this is the case we usually download from github instead.
They can be run by adding `pytest` to `buildInputs` and override checkPhase:

```
checkPhase = "py.test .";
```